apps/web/pages/api/auth/[...nextauth].tsx

```diff
@@ -316,13 +359,16 @@ export default NextAuth({
           return await autoMergeIdentities();
         }
 
+        const metadata = userMetadata.parse(existingUser.metadata);
+
         return {
           ...token,
           id: existingUser.id,
           name: existingUser.name,
           username: existingUser.username,
           email: existingUser.email,
           role: existingUser.role,
+          sessionTimeout: metadata?.sessionTimeout,
```

Comment:
Making sure we have the `sessionTimeout` value to use it when encoding the session token